paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/sink/cdc/MultiTableUpdatedDataFieldsProcessFunction.java

```diff
@@ -53,16 +47,17 @@
  * be 1.
  */
 public class MultiTableUpdatedDataFieldsProcessFunction
-        extends ProcessFunction<Tuple2<Identifier, List<DataField>>, Void> {
+        extends UpdatedDataFieldsBaseProcessFunction<Tuple2<Identifier, List<DataField>>, Void> {
 
     private static final Logger LOG =
             LoggerFactory.getLogger(MultiTableUpdatedDataFieldsProcessFunction.class);
 
     private final Map<Identifier, SchemaManager> schemaManagers = new HashMap<>();
-    private Catalog catalog;
+
     private final Catalog.Loader catalogLoader;
 
     public MultiTableUpdatedDataFieldsProcessFunction(Catalog.Loader catalogLoader) {
+        super(catalogLoader);
         this.catalogLoader = catalogLoader;
```

Comment:
delete `this.catalogLoader = catalogLoader`

Reply:
thanks for your suggestion, addressed it